ishell.go

```diff
@@ -495,31 +496,44 @@ func (s *Shell) multiChoice(options []string, text string, init []int, multiResu
 				selected = toggle(selected, cur)
 			}
 		}
-		update()
+		select {
```

Comment:
It is async sending to channel. If this channel already has element (previous refresh) it will be blocked.